src/parsers/texttracks/ttml/html/parse_ttml_to_div.ts

```diff
@@ -35,33 +35,87 @@ import parseCue, {
-                            "opacity",
-                            "origin",
-                            "overflow",
-                            "padding",
-                            "textAlign",
-                            "textDecoration",
-                            "textOutline",
-                            "unicodeBidi",
-                            "visibility",
-                            "wrapOption",
-                            "writingMode",
-
-                            // Not managed anywhere for now
-                            // "showBackground",
-                            // "zIndex",
-                            ];
+const STYLE_ATTRIBUTES = [ "backgroundColor",
+                           "color",
+                           "direction",
+                           "display",
+                           "displayAlign",
+                           "extent",
+                           "fontFamily",
+                           "fontSize",
+                           "fontStyle",
+                           "fontWeight",
+                           "lineHeight",
+                           "opacity",
+                           "origin",
+                           "overflow",
+                           "padding",
+                           "textAlign",
+                           "textDecoration",
+                           "textOutline",
+                           "unicodeBidi",
+                           "visibility",
+                           "wrapOption",
+                           "writingMode",
+
+                           // Not managed anywhere for now
+                           // "showBackground",
+                           // "zIndex",
+                           ];
+
+/**
+ * Apply a default style to TTML cue if no style has been already
+ * declared and no conflict is detected with current cue style.
+ *
+ * No position, orientation and dimension style should have been
+ * set to avoid any conflict.
+ *
+ * The default style propose to set the cue at the bottom, centered
+ * and lightly spaced apart from the edges :
+ *
+ *        -----------------------------------------------
+ *        |                                             |
+ *        |                                             |
+ *        |                                             |
+ *        |                                             |
+ *        |                                             |
+ *        |                                             |
+ *        |            subtitle is displayed            |
+ *        |                    here                     |
+ *        -----------------------------------------------
+ *
+ * @param {Object} cue
+ */
+function applyDefaultTTMLCSSStyle(cue: ITTMLHTMLCue): void {
+  const { element } = cue;
+  const { style: cueStyle } = element;
+  const { childNodes: paragraphNodes } = element;
+  const mainParagraphNode = paragraphNodes[0] as HTMLElement;
```

Comment:
Why are we checking only for the first and setting only for the first?

Another thing we have to be careful about is that we now could be breaking the style by changing code at another place.

This could be improved. Maybe we could have a boolean somewhere called `shouldApplyDefaultStyle` or something like that, passed to every functions handling style, that would be set to `false` by any code applying the corresponding style.
Maybe we could also refacto the style handling in the TTML code so it all happens in a single place.